Remove try catch in test_torchinductor

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #93005
* __->__ #93004
* #93003

I think this was holdover compat code from multiple repros. we should error on failure. 

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

eellison committed:
Remove try catch in test_torchinductor

[ghstack-poisoned]

eellison committed:
Update on "Remove try catch in test_torchinductor"


I think this was holdover compat code from multiple repros. we should error on failure. 

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

ngimel commented on test/inductor/test_torchinductor.py:
```diff
@@ -35,42 +37,34 @@
+from torch._inductor.utils import has_torchvision_roi_align, timed
+from torch.fx.experimental.symbolic_shapes import FloorDiv
+
+# This will only pass on pytorch builds newer than roughly 5/15/2022
```
nit: I think we can remove this

eellison committed:
Update on "Remove try catch in test_torchinductor"


I think this was holdover compat code from multiple repros. we should error on failure. 

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

eellison committed:
Update on "Remove try catch in test_torchinductor"


I think this was holdover compat code from multiple repros. we should error on failure. 

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

eellison commented:
Looks like windows fails when we remove the try catch...

ngimel commented:
We can skip tests on windows for now, it's a pre-existing condition?

eellison committed:
Update on "Remove try catch in test_torchinductor"


I think this was holdover compat code from multiple repros. we should error on failure. 

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

eellison commented:
@pytorchbot merge -f "unrelated failure"
